apps/root/src/pages/transfer/transfer-container/components/transfer-form/index.tsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import styled from 'styled-components';
+import { Paper, Typography } from 'ui-library';
+import NetworkSelector from '../network-selector';
+import WalletSelector from '../wallet-selector';
+import TokenSelector from '../token-selector';
+import RecipientAddress from '../recipient-address';
+import TransferButton from '../transfer-button';
+import useActiveWallet from '@hooks/useActiveWallet';
+import { FormattedMessage } from 'react-intl';
+import { useParams } from 'react-router-dom';
+
+const StyledPaper = styled(Paper)`
+  margin-top: 16px;
+  padding: 16px;
+  position: relative;
+  overflow: hidden;
+  border-radius: 20px;
+  flex-grow: 1;
+  background-color: rgba(255, 255, 255, 0.01);
+  backdrop-filter: blur(6px);
+`;
+
+const TransferForm = () => {
+  const {
```

Comment:
Maybe we should set all the parameters here, so we don't have to prop drill them to the other components, and they stay a bit more abstracted